web_src/js/features/repo-projects.js

```diff
@@ -2,31 +2,35 @@ import $ from 'jquery';
 import {useLightTextOnBackground} from '../utils/color.js';
 import tinycolor from 'tinycolor2';
 import {createSortable} from '../modules/sortable.js';
-
-const {csrfToken} = window.config;
+import {POST, DELETE, PUT} from '../modules/fetch.js';
 
 function updateIssueCount(cards) {
   const parent = cards.parentElement;
   const cnt = parent.getElementsByClassName('issue-card').length;
   parent.getElementsByClassName('project-column-issue-count')[0].textContent = cnt;
 }
 
-function createNewColumn(url, columnTitle, projectColorInput) {
-  $.ajax({
-    url,
-    data: JSON.stringify({title: columnTitle.val(), color: projectColorInput.val()}),
-    headers: {
-      'X-Csrf-Token': csrfToken,
-    },
-    contentType: 'application/json',
-    method: 'POST',
-  }).done(() => {
+async function createNewColumn(url, columnTitle, projectColorInput) {
+  try {
+    const response = await POST(url, {
+      data: {
+        title: columnTitle.val(),
+        color: projectColorInput.val(),
+      },
+    });
+
+    if (!response.ok) {
+      throw new Error('Failed to create new column');
+    }
+  } catch (error) {
+    console.error(error);
+  } finally {
     columnTitle.closest('form').removeClass('dirty');
     window.location.reload();
```

Comment:
I wouldn't reload the page in error case. This error sounds like one that should show a toast.

Reply:
This PR isn't meant to change any functionality, it's out of scope. This is purely to remove jQuery AJAX.

Reply:
Yeah I understand it, but I doubt the benefit of logging an error and then reloading the page. Browser console is usually set up so it clears on reload. So maybe just don't log it at all if that was the previous behaviour.

Reply:
https://github.com/go-gitea/gitea/pull/29814/commits/0520d2f6c5c1b2c87fe0b4abfec9f1da1acf76cd